docs/EMBEDDING.md

```diff
@@ -0,0 +1,49 @@
+# Embedding and Deploying noVNC Application
+
+This document describes how to embed and deploy the noVNC application, which
+includes settings and a full user interface. If you are looking for
+documentation on how to use the core noVNC library in your own application,
+then please see our [library documentation](LIBRARY.md).
+
+## Files
+
+The noVNC application consists of the following files and directories:
+
+* `vnc.html` - The main page for the application and where users should go. It
+  is possible to rename this file.
+
+* `app/` - Support files for the application. Contains code, images, styles and
+  translations.
+
+* `core/` - The core noVNC library.
+
+* `vendor/` - Third party support libraries used by the application and the
+  core library.
+
+The most basic deployment consists of simply serving these files from a web
+server and setting up a WebSocket proxy to the VNC server.
+
+## Parameters
+
+The noVNC application can be controlled by including certain settings in the
+query string. Currently the following options are available:
+
+FIXME
```

Comment:
Is this work in progress or are you fine leaving this FIXME until after the release?

Reply:
Not sure. :)

I mainly wanted to get the PR up before the weekend so that @DirectXMan12 could see I started looking at #961 and friends.

But it would be nice to have that section filled in as well.